app/models/user.rb

```diff
@@ -107,6 +107,14 @@ def mhv_account_type
     identity.mhv_account_type || MhvAccountTypeService.new(self).mhv_account_type
   end
 
+  def mhv_account_state
+    return 'DEACTIVATED' if (va_profile.mhv_ids.to_a - va_profile.active_mhv_ids.to_a).any?
+    return 'MULTIPLE' if va_profile.active_mhv_ids.size > 1
+    return 'NONE' if mhv_correlation_id.blank?
```

Comment:
Wouldn't it be possible for all of these scenarios to evaluate to `true`? - in which case, is it correct to favor `'DEACTIVATED'` just because it's the first guard clause?

(disclaimer: i know very little about MHV)

Reply:
Yes it's possible, at least for DEACTIVATED and MULTIPLE to both be true. The remediation is the same - need to contact the help desk and get your accounts straightened out, so I think preferring DEACTIVATED is fine.

Reply:
would `mhv_ids` or `active_mhv_ids` ever be nil?

Reply:
I don't see how - they are populated from an Array.select which should always return an empty or populated array.